tools/scripts/publish.plugin.ts

```diff
@@ -1,47 +0,0 @@
-import { CreateNodes, CreateNodesContext, readJsonFile } from '@nx/devkit';
-import { dirname, join } from 'node:path';
-import { ProjectConfiguration } from 'nx/src/config/workspace-json-project-json';
-import { PackageJson } from 'nx/src/utils/package-json';
-
-export const createNodes: CreateNodes = [
-  '**/project.json',
-  (projectConfigurationFile: string, opts, context: CreateNodesContext) => {
-    const root = dirname(projectConfigurationFile);
-    const projectConfiguration: ProjectConfiguration = readJsonFile(
-      projectConfigurationFile,
-    );
-
-    const isPublishable = Boolean(projectConfiguration?.targets?.publish);
-
-    if (!isPublishable) {
-      return {};
-    }
-
-    return {
-      projects: {
-        [root]: {
-          targets: publishTargets(projectConfiguration, root),
-        },
-      },
-    };
-  },
-];
-
-function publishTargets(projectConfig: ProjectConfiguration, root: string) {
-  const { name: projectName } = projectConfig;
-  const { name: packageName } = readJsonFile<PackageJson>(
-    join(root, 'package.json'),
-  );
-  return {
-    publish: {
-      command: `node tools/scripts/publish.mjs --name=${projectName} --ver={args.ver} --tag={args.tag}`,
-      dependsOn: ['build'],
-    },
```

Comment:
This target life's now in [`tools/src/publish/publish.plugin.ts`](https://github.com/code-pushup/cli/pull/798/files#diff-0a626ed9ff7166799fbd4373a4bc8812e2d2b4fb7539a00e4a8c0c603c97a413)